sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateAsyncClient.java

```diff
@@ -797,7 +803,7 @@ private CertificatePolicy getDefaultPolicy() {
      */
     private Mono<PagedResponse<CertificateProperties>> listCertificatesNextPage(String continuationToken, Context context) {
```

Comment:
[Here](https://github.com/Azure/azure-sdk-for-java/pull/6136/files#diff-e893e8c436eef93e3b0d93d24f0620a2L793), it should be `listPropertiesOfCertificates`